espnet2/tasks/mt.py

```diff
@@ -442,6 +444,7 @@ def build_model(cls, args: argparse.Namespace) -> ESPnetMTModel:
             decoder=decoder,
             token_list=token_list,
             src_token_list=src_token_list,
+            hubert=None,
```

Comment:
no need for this

Reply:
I tried deleting the variable but got an error.

Reply:
you need to delete the line in discrete_asr_espnet_model.py

Reply:
at the same time